openwrt-relayd-cve-2020-11752/README.md

```diff
@@ -0,0 +1,47 @@
+# OpenWRT mdns CVE-2020-11752
+
+Three out-of-bounds access issues were found in OpenWRT's relayd. They were reported to the OpenWRT security address on April 9 2020, and a fix ([1](https://git.openwrt.org/?p=project/relayd.git;a=commit;h=796da66abd074037f3b48475772d65874cc6de54), [2](https://git.openwrt.org/?p=project/relayd.git;a=commit;h=f4d759be54ceb37714e9a6ca320d5b50c95e9ce9)) was released soon after.
+
```

Comment:
nice!  Couple of requests:
*  Please hard-wrap at <80 char line limit.
* Add a link to the advisory https://openwrt.org/advisory/2020-05-06-2